snapcraft/extensions/kde_neon_6.py

```diff
@@ -309,5 +388,6 @@ def get_parts_snippet(self) -> Dict[str, Any]:
                 "source": str(source),
                 "plugin": "make",
                 "make-parameters": [f"PLATFORM_PLUG={self.kde_snaps.content_kf6}"],
```

Comment:
This parameter isn't used anymore, from core22 onwards, but I can make use of it.

Reply:
Done in #5109